Golem/Yagna/Model/ActivityStatePair.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Golem.Model
+{
+    public class ActivityStatePair
+    {
+        public List<ActivityState.StateType?> state { get; set; }
+
```

Comment:
Github is telling that this will generate warnings. Please fix them, because the number of warnings is growing

Reply:
Fixed it and few others.